templates/home/home-fr.html

```diff
@@ -14,16 +14,17 @@ <h1 property="name" id="wb-cont">Canada.ca</h1>
 			<p class="mrgn-tp-md">Le site officiel du gouvernement du Canada</p>
 		</div>
 	</div>
-	<div class="provisional gc-featured-link">
+	<section class="provisional gc-featured-link">
 		<div class="container">
 			<div class="row">
 				<div class="col-xs-12">
-					<p>En vedette&nbsp;:&nbsp;</p>
+					<h2 class="wb-inv">Lien en vedette</h2>
```

Comment:
Je crois que ceci reflète mieux l'anglais :
```suggestion
					<h2 class="wb-inv">En vedette</h2>
```